### PR TITLE
Feat/s3_admin-api-unactive-user_OU-53_DQuyen

### PR DESCRIPTION
**Pull request by quyenduc26**
Create the modify user API instead of creating an inactive user API only

### CODE REVIEW

File: src/user/dto/update-user.dto.ts [hunk 1]
```diff
@@ -1,14 +1,10 @@
 import { PartialType } from '@nestjs/swagger';
+// import { PartialType } from '@nestjs/mapped-types';
```
Comment (NhatNguyen1502) [inline src/user/dto/update-user.dto.ts]: /why you comment this line?

File: src/user/user.controller.ts [hunk 2]
```diff
@@ -16,6 +16,15 @@ export class UserController {
   create(@Body() createUserDto: CreateUserDto) {
     return this.userService.create(createUserDto);
   } 
+  
+  @Patch(':id')
+  @ApiOperation({ summary: 'Modify user' })  
+  @ApiCreatedResponse({ description: 'User status has been successfully updated' }) 
+  @ApiBadRequestResponse({ description: 'Bad request. Check your input data' }) 
+  update(@Param('id') id: string, @Body() updateUserDto: UpdateUserDto) {
+    // console.log(updateUserDto)
```
Comment (NhatNguyen1502) [inline src/user/user.controller.ts]: delete this log

File: src/user/user.controller.ts [hunk 3]
```diff
@@ -16,6 +16,15 @@ export class UserController {
...
+    return this.userService.update(+id, updateUserDto);
+  }
```
Comment (NhatNguyen1502) [inline src/user/user.controller.ts:26]: Is plus operator necessary?

Comment (quyenduc26) [reply src/user/user.controller.ts:26]: The 'id' parameter is a string; adding the plus operator converts it to a number.

File: src/user/user.controller.ts (hunk 2 above)
Comment (quyenduc26) [reply src/user/user.controller.ts]: ok bro

File: src/user/dto/update-user.dto.ts (hunk 1 above)
Comment (quyenduc26) [reply src/user/dto/update-user.dto.ts]: I forgot to delete it

**Review by NhatNguyen1502 — APPROVED** [review]

**Pull request merged**